### PR TITLE
chore: remove docs validation for ts sdk api docs

### PR DESCRIPTION
**Pull request by Dhaiwat10**
This PR removes the validation in place for the TS SDK API docs since they have now been isolated in a different project on a different URL (https://github.com/FuelLabs/fuels-ts/pull/3237) and they are now only referenced from the 'main' TS SDK docs.

### CODE REVIEW

**Review by helciofranco — APPROVED** [review]

File: docs-hub/vp-docs.js [hunk 1]
```diff
@@ -94,29 +91,6 @@ function checkForUnusedFiles(srcFolderPath, subfolders) {
     const folderPath = path.join(srcFolderPath, folder);
     const subfolderNames = fs.readdirSync(folderPath);
     const parentFolder = folderPath.split("/").pop();
-    subfolderNames.forEach((subFile) => {
```
Comment (petertonysmith94) [inline docs-hub/vp-docs.js:97]: @Dhaiwat10 Why was this removed?
